server/guiserver/apps.py

```diff
@@ -108,7 +108,7 @@ def server():
         'jujugui.gzip': options.gzip,
         'jujugui.insecure': options.insecure,
         'jujugui.interactive_login': options.interactivelogin,
-        'jujugui.jem_url': options.jemurl,
+        'jujugui.bundleservice_url': options.bundleservice_url,
```

Comment:
'jujugui.bundleservice_url' is read by options.py and converted into the bundleServiceURL value that you see in config.js -- config.js is the route that is served by the config view in views.py.